### PR TITLE
Connection to database 3

### PR DESCRIPTION
**Pull request by Pliama**

### CODE REVIEW

File: src/database/database.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+
+// DSN returns the Data Source Name
+func DSN(ci PostgreSQLInfo) string {
+	return fmt.Sprintf("host=%s port=%d user=%s "+
+		"password=%s dbname=%s sslmode=disable",
+		ci.Hostname, ci.Port, ci.Username, ci.Password, ci.DatabaseName)
+}
+func DSN_Redis(ci RedisInfo) string {
+	return fmt.Sprintf("%s"+":"+"%d",ci.URL,ci.Port)
```
Comment (Nastya-Kruglikova) [inline src/database/database.go]: Maybe fmt.Sprintf("%s:%d",ci.URL,ci.Port) will work ?

File: src/database/database.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+
+// Connect to the database
+func Connect(d Info){
+	var err error
+
+	// Store the config
+	databases = d
+
+	for index,_:= range d.Type[:]{
```
Comment (Nastya-Kruglikova) [inline src/database/database.go]: You can write just 
for index := range d.Type {

Comment (Nastya-Kruglikova) [reply src/database/database.go]: Also `i` is good enought for index and pretty standart

File: src/database/database.go [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+		switch d.Type[index] {
+		case TypePostgreSQL:
```
Comment (Nastya-Kruglikova) [inline src/database/database.go]: It is better to make two types DB and Cache and its methods Connect instead of switch

File: src/main.go [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+	configFile := flag.String("config", "./config.json", "Configuration file in JSON-format")
+	flag.Parse()
+
+	if len(*configFile) > 0 {
+		config.FilePath = *configFile
+	}
+
+	err := config.Load()
+
+	database.Connect(config.Config.Database)
```
Comment (Nastya-Kruglikova) [inline src/main.go]: Could there be any errors?

File: src/main.go [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+
+	if err != nil {
```
Comment (Nastya-Kruglikova) [inline src/main.go]: This is related to config.Load(), move it to correct place

File: src/config/config.go [hunk 6]
```diff
@@ -18,6 +19,7 @@ var (
 type Configuration struct {
 	ListenURL   string `json:"ListenURL"`
 	LogFilePath string `json:"LogFilePath"`
+	Database  database.Info   `json:"Database"`
```
Comment (mshto) [inline src/config/config.go]: Could you use refactoring tools in your IDE?

**Pull request closed without merging**